docs/user_guide/capsules/isolation.md

```diff
@@ -0,0 +1,37 @@
+---
```

Comment:
The image references 'isolated host' but throughout the term client is used. Might consider making those consistent.